src/backend/data/user.js

```diff
@@ -0,0 +1,30 @@
+const Feed = require('./feed.js');
+
+class User {
+  constructor(name, email, id) {
+    this.name = name;
+    this.email = email;
+    this.id = id;
+  }
```

Comment:
Our user schema should look like this:
https://github.com/Seneca-CDOT/telescope/blob/b48c614d0e84b4e5bc8036938b7104821861eb94/src/backend/web/__mocks__/authentication.js#L14-L18

- We need an `isAdmin` data member.
- We should not have `id` as a constructor parameter. Instead, we should initialize user IDs by hashing the given email.

Reply:
The layer that they are working at is lower down than what you're describing.  The `isAdmin` stuff happens higher up.  Also, they need to accept the `id` as it comes from passport.